### DISCUSSION TITLE
How to use `ClosestFeature()` to get two or more features adjacent to one da peaks?

### DISCUSSION THREAD

**Discussion by jiangpuxuan** [stuart-lab/signac#1048] [Q&A] [answered]
Here is a typical situation: I have a da peaks and it is adjecent to two genes. The first gene is the closetest, but the second gene has much more experimental support. How can I fetch both the two gene by `ClosestFeature()`?
![sss](https://user-images.githubusercontent.com/81287604/158821497-498140b6-250e-447c-ad3b-e2fcd08cb26d.png)

**Comment by timoast** [comment, marked as answer]
The `ClosestFeature()` function in Signac will only report the closest feature. Do run a more detailed analysis, you can use functions in other packages, like the GenomicRanges package: https://bioconductor.org/packages/GenomicRanges/

**Reply by jiangpuxuan** [reply]
> The `ClosestFeature()` function in Signac will only report the closest feature. Do run a more detailed analysis, you can use functions in other packages, like the GenomicRanges package: https://bioconductor.org/packages/GenomicRanges/
Thank you for your help.
